src/engine/source/kvdb2/include/kvdb2/kvdbScope.hpp

```diff
@@ -0,0 +1,32 @@
+#ifndef _KVDBSCOPE_H
```

Comment:
I think it should be `_KVDB_SCOPE_H`

Reply:
Will be fixed in later documentation step.